pkg/apis/operator/v1alpha1/common.go

```diff
@@ -59,6 +59,8 @@ type KComponentSpec interface {
 	GetResources() []ResourceRequirementsOverride
 	// GetVersion gets the version to be installed
 	GetVersion() string
+	// GetManifests gets the list of manifests to be installed
```

Comment:
Is this redundant with GetVersion? Can they ever disagree?

Reply:
I was just going to ask how/whether version interacts with this field. :grin:

Reply:
Any update here?